lib/model/model.go

```diff
@@ -69,6 +69,12 @@ type Availability struct {
 	FromTemporary bool              `json:"fromTemporary"`
 }
 
+type DirectoryTree struct {
```

Comment:
DirectoryTree.IsDirectory = false is somewhat a misleading name.

As DirectoryTree is sometimes a file.

FolderItem or just Item?

Reply:
Good point. Will rename.